src/lexer.coffee

```diff
@@ -1314,6 +1314,8 @@ OPERATOR   = /// ^ (
 
 WHITESPACE = /^[^\n\S]+/
 
+exports.LINEBREAK = LINEBREAK = /\r\n|[\r\n\u2028\u2029]/ug
```

Comment:
This regex came from [here](https://github.com/eslint/eslint/blob/master/lib/shared/ast-utils.js#L12)